### DISCUSSION TITLE
IBM i Output option is not in the list of options on the Output panel

### DISCUSSION THREAD

**Discussion by Markj0616** [codefori/vscode-ibmi#1634] [Q&A] [answered]
In documentation and videos, there is an option on the Output panel to display the spool files when a program is compiled through VS Code. I cannot find this option. I've selected all of the other IBM i options in hopes that the name just changed, but none of them displays my compile listing. Is there another extension that i need to install or something?

**Comment by sebjulliand** [comment, marked as answer]
There has been a change in the way actions are run since videos were made.
Actions output are now found under the Terminal tab. When an action is done, a notification will also appear to ask if you want to open and display the output.
![image](https://github.com/codefori/vscode-ibmi/assets/11096890/334bd899-438d-41b2-94c2-edbffaa1ddae)

**Reply by Markj0616** [reply]
I guess there is something here that is linked to my issue with the compile process. The CRTBNDRPG command isn't creating objects on my IBM i even if they pass the compile (will compile on the 5250 screen, but not from VS Code. This is what i get when i open the output for my compile 

Current library: QGPL
Library list: QS36F DESTINY DESTINY_R OE OEF IM IMF AP APF GL GLF AR SY SYF QTEMP #LIBRARY MARK QGPL RVILIB
Commands:
	CRTBNDRPG PGM(MARK/LOADMT) SRCFILE(MARK/OESRC) OPTION(*EVENTF) DBGVIEW(*SOURCE) TGTRLS(*CURRENT)


Fetching errors for MARK/LOADMT.

**Reply by worksofliam** [reply]
Please share the contents of your Code for IBM i output channel after the list doesn't appear.

<img width="939" alt="image" src="https://user-images.githubusercontent.com/3708366/178739532-03c8ad30-84de-4698-9904-65938f2ef651.png">

**Comment by worksofliam** [comment]
Yes, we removed the 'IBM i Output channel' in place of two things:

* The Terminals tab will show the compile listing post build
* You will see a button to 'open output' post complication

**Comment by Markj0616** [comment]
```

/home/MJOHNSON: /QOpenSys/usr/bin/qsh
liblist -d QS36F DESTINY DESTINY_R OE OEF IM IMF AP APF GL GLF AR SY SYF QTEMP #LIBRARY MARK QGPL RVILIB && liblist -c QGPL && liblist -a RVILIB QGPL MARK #LIBRARY QTEMP SYF SY AR GLF GL APF AP IMF IM OEF OE DESTINY_R DESTINY QS36F && system  "CRTBNDRPG PGM(MARK/LOADMT) SRCFILE(MARK/OESRC) OPTION(*EVENTF) DBGVIEW(*SOURCE) TGTRLS(*CURRENT)"; if [[ $? -ne 0 ]]; then exit 1; fi
{
    "code": 0,
    "signal": null,
    "stdout": "",
    "stderr": ""
}

/home/MJOHNSON: system "QSYS/CPYTOIMPF FROMFILE(MARK/EVFEVENT LOADMT) TOSTMF('/tmp/vscodetemp-O_W1bqhtkv') MBROPT(*REPLACE) STMFCCSID(1208) RCDDLM(*CRLF) DTAFMT(*DLM) RMVBLANK(*TRAILING) ADDCOLNAM(*SQL) FLDDLM(',') DECPNT(*PERIOD)"
{
    "code": 255,
    "signal": null,
    "stdout": "",
    "stderr": "CPF2803: To-file EVFEVENT in MARK not allowed.\nCPF2817: Copy command ended because of error."
}



```

**Reply by worksofliam** [reply]
What version of IBM i is this?

**Comment by Markj0616** [comment]
2.4.0

***@***.***

From: barry ***@***.***>
Sent: Tuesday, November 7, 2023 9:19 AM
To: codefori/vscode-ibmi ***@***.***>
Cc: Mark Johnson ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [codefori/vscode-ibmi] IBM i Output option is not in the list of options on the Output panel (Discussion #1634)


What version of IBM i is this?

-

**Reply by worksofliam** [reply]
This is the Code for IBM i version - what version of IBM i OS are you using?

**Comment by Markj0616** [comment]
Sorry ... V7R4M0


From: barry ***@***.***>
Sent: Tuesday, November 7, 2023 9:32 AM
To: codefori/vscode-ibmi ***@***.***>
Cc: Mark Johnson ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [codefori/vscode-ibmi] IBM i Output option is not in the list of options on the Output panel (Discussion #1634)


This is the Code for IBM i version - what version of IBM i OS are you using?

-
